src/mantine-core/src/MultiSelect/MultiSelect.styles.ts

```diff
@@ -27,16 +25,18 @@ export default createStyles((theme, { size, invalid }: MultiSelectStylesParams)
   },
 
   searchInput: {
+    ...theme.fn.fontStyles(),
     flex: 1,
     minWidth: 60,
     backgroundColor: 'transparent',
     border: 0,
     outline: 0,
     fontSize: theme.fn.size({ size, sizes: theme.fontSizes }),
     padding: 0,
-    margin: theme.spacing.xs / 2,
+    marginLeft: theme.spacing.xs / 2,
     appearance: 'none',
     color: 'inherit',
+    lineHeight: `${theme.fn.size({ size, sizes: INPUT_SIZES }) - 2}px`,
```

Comment:
This is the same way the line height is calculated in the [Input styles](https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Input/Input.styles.ts#L126)

Reply:
Will in work correctly is MultiSelect has multiple lines? It happens when there are a lot of items selected.

Reply:
Seems fine to me:

Chrome:
![image](https://user-images.githubusercontent.com/9464690/185344130-23261a89-5beb-4002-9f5d-9341922847e5.png)

Edge:
![image](https://user-images.githubusercontent.com/9464690/185344217-4dd568e8-56bf-40f8-abea-41ee86670103.png)